src/edu/wright/cs/raiderplanner/controller/StudyPlannerController.java

```diff
@@ -160,37 +168,93 @@ public boolean createStudyProfile(HubFile hubFile) {
+				Calendar temp = Calendar.getInstance();
+				temp.add(Calendar.HOUR_OF_DAY, hours2);
+				Date date = temp.getTime();
+				System.out.print(date);
+				if (date.after(task.getDeadlineDate())) {
+					Notification not = new Notification("Assignment due in two days!",
+									new GregorianCalendar(), task.getName(), task);
+					MainController.getSpc().getPlanner().addNotification(not);
+					task.toggleTwoDayNotification();
+				}
+			}
+		}
+		/**Was the original commented out code, keeping for now for future reference if needed.
+		 * for (Map.Entry<ModelEntity, boolean[]> entry : this.planner
+		 * 						.getDeadlineNotifications().entrySet()) {
+		 *	System.out.print(entry.getKey());
+		 *	if (entry.getKey() instanceof Assignment) {
+		 *		if (!entry.getValue()[0]) {
+		 *			GregorianCalendar temp = new GregorianCalendar();
+		 *			temp.add(Calendar.HOUR, -hours1);
+		 *			Date date = temp.getTime();
+		 *			if (entry.getKey() instanceof Coursework) {
+		 *				if (date.after((((Coursework) entry.getKey()).getDeadline().getDate()))) {
+		 *					Notification not = new Notification("Assignment due in a week!",
+		 *									new GregorianCalendar(), entry.getKey().getName(),
+		 *									entry.getKey());
+		 *		 			MainController.getSpc().getPlanner().addNotification(not);
+		 *					entry.getValue()[0] = true;
+		 *				}
+		 *			}
+		 *			if (entry.getKey() instanceof Exam) {
+		 *				if (date.after((((Exam) entry.getKey()).getTimeSlot().getDate()))) {
+		 *					Notification not = new Notification("You have an exam in a week!",
+		 *										new GregorianCalendar(), entry.getKey().getName(),
+		 *										entry.getKey());
+		 *		 			MainController.getSpc().getPlanner().addNotification(not); entry
+		 *										.getValue()[0] = true;
+		 *				}
+		 *		 	}
+		 *		}
+		 *		else if (!entry.getValue()[1]) {
+		 *			GregorianCalendar temp = new GregorianCalendar();
+		 *			temp.add(Calendar.HOUR, -hours2);
+		 *			Date date = temp.getTime();
+		 *			if (entry.getKey() instanceof Coursework) {
+		 *				if (date.after((((Coursework) entry.getKey()).getDeadline().getDate()))) {
+		 *					Notification not = new Notification("Assignment due in a two days!",
+		 *									new GregorianCalendar(), entry.getKey().getName(),
+		 *									entry.getKey());
+		 *		 			MainController.getSpc().getPlanner().addNotification(not); entry
+		 *									.getValue()[1] = true;
+		 *				}
+		 *			}
+		 *			if (entry.getKey() instanceof Exam) {
+		 *				if (date.after((((Exam) entry.getKey()).getTimeSlot().getDate()))) {
+		 *					Notification not = new Notification("You have an exam in two days!",
+		 *									new GregorianCalendar(), entry.getKey().getName(),
+		 *									entry.getKey());
+		 *					MainController.getSpc().getPlanner().addNotification(not); entry
+		 *									.getValue()[1] = true;
+		 *				}
+		 *		 	}
+		 *		}
+		 *		else this.planner.getDeadlineNotifications().remove(entry);
+		 *	}
+		 *}
+		*/
```

Comment:
You really don't need to keep the old code in a comment. Git will automatically track the changes and if someone would like to see what changed in the old code, they can review the commits.